maquetado/scss/tech.scss

```
@@ -1,31 +1,8 @@
 .tech {
-  align-items: center;
-  background-color: $green;
-  border-radius: 8px;
-  display: flex;
-  flex-direction: column;
-  justify-items: center;
-  margin: 16px;
-  max-width: 208px;
-  padding: 16px;
-  position: relative;
-  width: 100%;
-
-  &::after {
-    border: 2px solid $green;
-    border-radius: 8px;
-    content: '';
-    height: 100%;
-    position: absolute;
-    transform: translate(-10px, -25px);
-    transition: 0.5s;
-    width: 100%;
-    z-index: -1;
-  }
-
-  &:hover::after {
-    transform: translate(8px,-8px);
-  }
+  display: grid;
+  grid-template-columns: 1fr;
```

Why this? I think that is better if you define in the container
```suggestion
  grid-template-columns: repeat(3, 1fr);
```